src/filters.js

```diff
@@ -3,6 +3,21 @@
 var lib = require('./lib');
 var r = require('./runtime');
 
+
+var normalize = function(value, defaultValue) {
+
+    if (value === null || value === undefined) {
+        return defaultValue;
+    }
+
+    if (value instanceof r.SafeString) {
```

Comment:
Please remove this block. The `normalize` function should only look like this:

```js
function normalize(v, default) {
  if(v === null || v === undefined || v === false) {
    return default;
  }
  return v;
}
```

The filters should work with a SafeString, unless something is amiss.